src/EnergyPlus/Psychrometrics.hh

```diff
@@ -105,58 +105,57 @@ namespace Psychrometrics {
     // Data
     // MODULE PARAMETER DEFINITIONS:
     // call for recurring errors
-    constexpr int iPsyTdpFnTdbTwbPb = 1;
-    constexpr int iPsyRhFnTdbWPb = 2;
-    constexpr int iPsyTwbFnTdbWPb = 3;
-    constexpr int iPsyTwbFnTdbWPb2 = 14;
-    constexpr int iPsyTwbFnTdbWPb3 = 15; // convergence
-    constexpr int iPsyVFnTdbWPb = 4;
-    constexpr int iPsyWFnTdpPb = 5;
-    constexpr int iPsyWFnTdbH = 6;
-    constexpr int iPsyWFnTdbTwbPb = 7;
-    constexpr int iPsyWFnTdbTwbPb2 = 16;
-    constexpr int iPsyWFnTdbRhPb = 8;
-    constexpr int iPsyPsatFnTemp = 9;
-    constexpr int iPsyTsatFnHPb = 10;
-    constexpr int iPsyTsatFnPb = 11;
-    constexpr int iPsyTsatFnPb2 = 17; // iterations
-    constexpr int iPsyRhFnTdbRhov = 12;
-    constexpr int iPsyRhFnTdbRhovLBnd0C = 13;
-    constexpr int iPsyTwbFnTdbWPb_cache = 18;
-    constexpr int iPsyPsatFnTemp_cache = 19;
 
-#ifdef EP_psych_stats
-    extern Array1D_string const PsyRoutineNames; // 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 - HR | 15 - max iter | 16 - HR | 17 -
-                                                 // max iter | 18 - PsyTwbFnTdbWPb_raw (raw calc) | 19 - PsyPsatFnTemp_raw (raw calc)
-
-    extern Array1D_bool const PsyReportIt; // PsyTdpFnTdbTwbPb     1 | PsyRhFnTdbWPb        2 | PsyTwbFnTdbWPb       3 | PsyVFnTdbWPb         4 |
-                                           // PsyWFnTdpPb          5 | PsyWFnTdbH           6 | PsyWFnTdbTwbPb       7 | PsyWFnTdbRhPb        8 |
-                                           // PsyPsatFnTemp        9 | PsyTsatFnHPb         10 | PsyTsatFnPb          11 | PsyRhFnTdbRhov       12 |
-                                           // PsyRhFnTdbRhovLBnd0C 13 | PsyTwbFnTdbWPb       14 - HR | PsyTwbFnTdbWPb       15 - max iter |
-                                           // PsyWFnTdbTwbPb       16 - HR | PsyTsatFnPb          17 - max iter | PsyTwbFnTdbWPb_cache 18 -
-                                           // PsyTwbFnTdbWPb_raw (raw calc) | PsyPsatFnTemp_cache  19 - PsyPsatFnTemp_raw (raw calc)
-#endif
-
-#ifndef EP_psych_errors
-#endif
+    constexpr std::array<std::string_view, static_cast<int>(PsychrometricFunction::Num)> PsyRoutineNames = {
+        "PsyTdpFnTdbTwbPb",
+        "PsyRhFnTdbWPb",
+        "PsyTwbFnTdbWPb",
+        "PsyVFnTdbWPb",
+        "PsyWFnTdpPb",
+        "PsyWFnTdbH",
+        "PsyWFnTdbTwbPb",
+        "PsyWFnTdbRhPb",
+        "PsyPsatFnTemp",
+        "PsyTsatFnHPb",
+        "PsyTsatFnPb",
+        "PsyRhFnTdbRhov",
+        "PsyRhFnTdbRhovLBnd0C",
+        "PsyTwbFnTdbWPb",
+        "PsyTwbFnTdbWPb",
+        "PsyWFnTdbTwbPb",
+        "PsyTsatFnPb",
+        "PsyTwbFnTdbWPb_cache",
+        "PsyPsatFnTemp_cache"}; // 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 |
```

Comment:
With this initialization, I wonder if the comment block at the end is really necessary.  I suggest either moving the comments up to each individual element, or delete the comment block if not necessary.